src/app/productboard/productboard.component.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  height: calc(100% + 2rem);
+  width: calc(100% + 4rem);
```

Comment:
Does this work with the flexible menu? Can't you use flex-1?

Reply:
(resolving this as discussed offline)